src/main/resources/org/sagebionetworks/web/client/view/AccountViewImpl.ui.xml

```diff
@@ -2,8 +2,6 @@
 <ui:UiBinder
   xmlns:ui="urn:ui:com.google.gwt.uibinder"
   xmlns:g="urn:import:com.google.gwt.user.client.ui"
-  xmlns:c="urn:import:com.google.gwt.user.cellview.client"
-  xmlns:s="urn:import:org.sagebionetworks.web.client.view"
```

Comment:
I optimized away imports for many of the XML files in the document. This made cmd+F for Bootstrap widgets much easier (many files included the namespace but did not use bootstrap widgets!)